recipes-bsp/device-tree/files/uz7ev-evcc-hdmi-v-n/system-bsp.dtsi

```diff
@@ -0,0 +1,220 @@
+#include "hdmi.dtsi"
+#include "include/dt-bindings/phy/phy.h"
+
```

Comment:
Looks like this file is similar to 'uz7ev-evcc-hdmi/system-bsp.dtsi'
So it is not needed, the build system will use uz7ev-evcc-hdmi/system-bsp.dtsi